Tools/scripts/docker_build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ $# -ne 1 ]
+  then
+    echo "No board supplied, run as ./docker_build.sh <board name> or ./docker_build.sh list"
+    exit 1
+fi
+
+BOARD=$1
+
+cd "$(git rev-parse --show-toplevel)"
+git submodule update --init --recursive
+
+PATH='/home/ardupilot/.local/bin:/usr/lib/ccache:/ardupilot/Tools/autotest:/opt/gcc-arm-none-eabi-6-2017-q2-update/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
```

Comment:
Hmm, this is an outdated line, it does only tack stuff onto PATH now.

How do you mean implicit? I added it here because otherwise `waf` stuff won't work in `docker run` unless you run `bash` and do things manually there.

Reply:
Oops, I forgot this when debugging, removed.